app/decorators/payment_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PaymentDecorator < Draper::Decorator
+  delegate_all
+
+  def amount
+    amount = read_attribute :amount
+    formatted_amount = "%.2f" % (amount.to_f/100)
+    "#{formatted_amount} €"
```

Comment:
fixed. I had to include  ActionView::Helpers::NumberHelper in the decorator to do this. Furthermore, it had to create a locale file.